### ISSUE TITLE
False positive when multiplying with int type struct field

### ISSUE THREAD

**Issue by ajpetersons** [charithe/durationcheck#7]
Struct fields are not taken into account when assessing duration multiplication. Consider a struct that has int field. When this field is multiplied by a valid duration value, linter throws an error (to multiply with duration, the field needs to be converted to time.Duration first).

See this example:
```go
package main

import (
	"time"
)

type s struct {
	d int
}

func main() {
	ss := s{10}
	_ = time.Duration(ss.d) * time.Second
}
```

**Comment by edigaryev** [comment]
This is also documented in the [time package](https://golang.org/pkg/time/) (see "To convert an integer number of units to a Duration, multiply:"):

```go
seconds := 10
fmt.Print(time.Duration(seconds)*time.Second) // prints 10s
```
